pkgs/racket-test-core/tests/racket/portlib.rktl

```diff
@@ -14,14 +14,29 @@
   (test '(#\h #\e #\l #\l #\o #\return #\newline #\t #\h #\e #\r #\e)
 	port->list read-char (p))
   (test "hello\r\nthere" port->string (p))
+  (test "hello\r\nthere" port->string/close (p))
```

Comment:
Oh, it's not defined there. Tests passed locally (run with `racket test.rkt` in this directory), so something's up. I'll make sure I'm actually running these tests.